README.md

```diff
@@ -3,18 +3,35 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T1055324)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-# FileManager for Web Forms - How to open PDF and Images in popup
-This example demonstrates how to open images and PDF-files using ASPxFileManager, ASPxPanel and ASPxImage controls.
+
+# The file manager for Web Forms - How to open PDF files and images in a popup window
+
+This example demonstrates how to use the [ASPxFileManager](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxFileManager), [ASPxPanel](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPanel?p=netframework), and [ASPxImage](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxImage) controls to open **PDF** files and images in a [popup window](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl?p=netframework).
+
 ![Sample](./Sample.png)
- ## Overview
-The [ASPxFileManager.SelectedFileOpened](https://docs.devexpress.com/AspNet/js-ASPxClientFileManager.SelectedFileOpened) event is handled to open the file using ASPxImage to show images or ASPxPanel with [Embed tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/embed) to show PDF inside a popup.
+
 ## Files to Look At
-[Default.aspx](./CS/OpenPdfOrImage/Default.aspx)
+
+* [Default.aspx](./CS/OpenPdfOrImage/Default.aspx) (VB: [Default.aspx](./VB/OpenPdfOrImage/Default.aspx))
+
+* [Default.aspx.cs](./CS/OpenPdfOrImage/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/OpenPdfOrImage/Default.aspx.vb))
+
+## Implementation Details
+
+The [ASPxFileManager](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxFileManager) control allows users to manage files. This example handles the [ASPxFileManager.SelectedFileOpened](https://docs.devexpress.com/AspNet/js-ASPxClientFileManager.SelectedFileOpened) event to determine the opened file's type. 
```

Comment:
The first sentence is too generic - it is more suitable for the class description than for the section titled Implementation Details. In this section, I expect text that is specific to this demo. In addition to that you end a paragraph with "determine the file type". This gives me the impression that this is all the handler does. 

```suggestion
The [ASPxFileManager](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxFileManager) displays a list of image and PDF files. When users attempt to open a file, the control raises the [ASPxFileManager.SelectedFileOpened](https://docs.devexpress.com/AspNet/js-ASPxClientFileManager.SelectedFileOpened) event. The handler determines the file type and shows the preview. 
```

Reply:
I'd like to put "PDF" in bold
```suggestion
The [ASPxFileManager](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxFileManager) displays a list of image and PDF files. When users attempt to open a file, the control raises the [ASPxFileManager.SelectedFileOpened](https://docs.devexpress.com/AspNet/js-ASPxClientFileManager.SelectedFileOpened) event. The handler determines the file type and shows the preview. 
```